<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="kola_price">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLabel" name="ice_tea">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ICE TEA</string>
      </property>
     </widget>
     <widget class="QLabel" name="americano_price">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>210</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_cola">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Əlavə et</string>
      </property>
     </widget>
     <widget class="QLabel" name="kola">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>COCA-COLA</string>
      </property>
     </widget>
     <widget class="QLabel" name="americano">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AMERICANO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_sprite">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Əlavə et</string>
      </property>
     </widget>
     <widget class="QLabel" name="ice_tea_price">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="pepsi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PEPSI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_ice_tea">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Əlavə et</string>
      </property>
     </widget>
     <widget class="QLabel" name="sprite_price">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLabel" name="sprite">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SPRITE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_pepsi">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Əlavə et</string>
      </property>
     </widget>
     <widget class="QLabel" name="pepsi_price">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>31</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Turqay Store</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_americano">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Əlavə et</string>
      </property>
     </widget>
     <widget class="QPushButton" name="go_to_basket">
      <property name="geometry">
       <rect>
        <x>164</x>
        <y>270</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>SƏBƏTƏ KEÇ</string>
      </property>
     </widget>
     <widget class="QLabel" name="empty">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>340</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ÖDƏNİŞ :</string>
      </property>
     </widget>
     <widget class="QLabel" name="americano_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TOTAL :</string>
      </property>
     </widget>
     <widget class="QLabel" name="pay_total_price">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="americano_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>AD :</string>
      </property>
     </widget>
     <widget class="QLabel" name="americano_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>KART NÖMRƏSİ :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_card_num">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>250</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="aproove_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>TƏSTİQLƏ</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="americano_3">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TOTAL :</string>
      </property>
     </widget>
     <widget class="QLabel" name="title_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SƏBƏT</string>
      </property>
     </widget>
     <widget class="QListWidget" name="bascet_list">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="total_price">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Script</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pay_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ÖDƏ</string>
      </property>
     </widget>
     <widget class="QPushButton" name="back_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>GERİ QAYIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="title_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>20</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>sağ olun</string>
      </property>
     </widget>
     <widget class="QPushButton" name="again_button">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>YENİDƏN BAŞLA</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>468</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
